Mobile Buy SDK/Mobile Buy SDK/Utils/BUYApplePayHelpers.m

```diff
@@ -51,11 +51,6 @@ @interface BUYApplePayHelpers ()
 
 @implementation BUYApplePayHelpers
 
-- (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout
-{
-	return [self initWithClient:client checkout:checkout shop:nil];
-}
-
 - (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout shop:(BUYShop *)shop
```

Comment:
Its not strictly necessary but I prefer to only expose mutable properties when they are required. As far as I can tell, there's never a need to set the `BUYCheckout` from the outside after initialization.